### PR TITLE
Hardware extensions bug fixes

### PR DESCRIPTION
**Pull request by evhan55**
### Resolves

- https://github.com/LLK/scratch-vm/issues/1566 - Set default motor power for EV3 and WeDo2 to 50%
  - WeDo 2.0 default = 100, block default = 100
  - EV3 default = 50, block default = 100
- https://github.com/LLK/scratch-vm/issues/1567 - EV3 motor position reporter gets inverted
- https://github.com/LLK/scratch-vm/issues/1538 - wedo2 "set motor direction" block sometimes stops the motor
- https://github.com/LLK/scratch-vm/issues/1364 - Microbit "button pressed?" block should return true/false
- Some minor cleanup

### CODE REVIEW

File: src/extensions/scratch3_wedo2/index.js [hunk 1]
```diff
@@ -676,19 +676,13 @@ class WeDo2 {
...
-                this.setLED(0x0000FF);
-            })
-            .then(() => { // TODO: Promise?
-                this._ble.startNotifications(
-                    BLEService.DEVICE_SERVICE,
-                    BLECharacteristic.ATTACHED_IO,
-                    this._onMessage
-                );
-            });
+        this.setLEDMode();
```
Comment (evhan55) [inline src/extensions/scratch3_wedo2/index.js:679]: Removing some Promises that didn't seem to do anything anymore.

File: src/extensions/scratch3_wedo2/index.js [hunk 2]
```diff
@@ -762,14 +756,12 @@ class WeDo2 {
...
 
-            this.send(BLECharacteristic.INPUT_COMMAND, cmd)
-                .then(() => { // TODO: Promise?
-                    this._ble.startNotifications(
-                        BLEService.IO_SERVICE,
-                        BLECharacteristic.INPUT_VALUES,
-                        this._onMessage
-                    );
-                });
+            this.send(BLECharacteristic.INPUT_COMMAND, cmd);
```
Comment (evhan55) [inline src/extensions/scratch3_wedo2/index.js:759]: Removing some Promises that didn't seem to do anything anymore.

File: src/extensions/scratch3_ev3/index.js [hunk 3]
```diff
@@ -683,17 +679,18 @@ class EV3 {
             // GET DEVICE LIST
             byteCommands[0] = Ev3Opcode.OPINPUT_DEVICE_LIST;
             byteCommands[1] = Ev3Value.NUM8; // 1 byte to follow
-            byteCommands[2] = 33; // 0x21 ARRAY // TODO: ????
-            byteCommands[3] = 96; // 0x60 CHANGED // TODO: ????
-            byteCommands[4] = 225; // 0xE1 size of global var - 1 byte to follow // TODO: ????
-            byteCommands[5] = 32; // 0x20 global var index "0" 0b00100000 // TODO: ????
+            byteCommands[2] = 33; // 0x21 ARRAY // TODO: document
```
Comment (evhan55) [inline src/extensions/scratch3_ev3/index.js:682]: I wanted to put in some reminders for someone to document some remaining EV3 hardware protocol things that didn't make it in the previous refactor.

Comment (evhan55) [reply src/extensions/scratch3_ev3/index.js:682]: https://github.com/LLK/scratch-vm/issues/1568

**Review by ericrosenbaum — APPROVED** [review]
LGTM!

**Pull request merged**